runtime/compiler/runtime/RelocationRuntime.cpp

```diff
@@ -133,24 +133,25 @@ const char *TR_RelocationRuntime::_reloErrorCodeNames[] =
    "svmValidationFailure",                             // 45
    "wkcValidationFailure",                             // 46
    "methodTracingValidationFailure",                   // 47
-
-   "classAddressRelocationFailure",                    // 48
-   "inlinedMethodRelocationFailure",                   // 49
-   "symbolFromManagerRelocationFailure",               // 50
-   "thunkRelocationFailure",                           // 51
-   "trampolineRelocationFailure",                      // 52
-   "picTrampolineRelocationFailure",                   // 53
-   "cacheFullRelocationFailure",                       // 54
-   "blockFrequencyRelocationFailure",                  // 55
-   "recompQueuedFlagRelocationFailure",                // 56
-   "debugCounterRelocationFailure",                    // 57
-   "directJNICallRelocationFailure",                   // 58
-   "ramMethodConstRelocationFailure",                  // 59
-   "catchBlockCounterRelocationFailure",               // 60
-   "staticDefaultValueInstanceRelocationFailure",      // 61
-
-   "maxRelocationError"                                // 62
-
+   "dynamicMethodFromcallsiteIndexValidationFailure",  // 48
+   "handleMethodFromcallsiteIndexValidationFailure" ,  // 49
```

Comment:
Come to think of it, I don't think these relocations can fail anyway. Maybe it's better just to assert and then these new error codes wouldn't be needed